apps/target-app/app.py

```diff
@@ -14,6 +14,8 @@
 
 file_path = Path(__file__).parent / "simulated-data.csv"
 
+ui.page_opts(title="Monitoring")
```

Comment:
Once you switch from `page_sidebar(ui.navset_bar())` to `page_navbar()` (where the page fn is implied by the structure of the Express app), you can use `ui.page_opts()` to set the page title.